test/persistence/user-tables/table-persistence-after-sinks.td.disabled

```diff
@@ -9,6 +9,9 @@
 
 > INSERT INTO sink_table VALUES (3)
 
+> SELECT COUNT(*) FROM sink_table
```

Comment:
? :thinking:

Reply:
I added this to make sure that the failure I was seeing below was with the sink itself and not the table implementation!

I think the best thing going forward is to actually re-enable this test with the `COUNT(*) FROM sink_table` and the kafka-verify below commented out until we address the TODO

Reply:
Actually -- this test is sink-specific so I think it's probably best to keep disabled until we do what's almost certainly a minor fix